editor/blocks/list/index.js

```diff
@@ -53,29 +57,57 @@ wp.blocks.registerBlock( 'core/list', {
 			onClick( attributes, setAttributes ) {
 				setAttributes( { align: 'justify' } );
 			}
+		},
+		{
+			icon: 'editor-ul',
+			title: wp.i18n.__( 'Convert to unordered' ),
+			isActive: ( { } ) => false, // To be implemented
```

Comment:
No issues here. We are probably in the habit of keeping interfaces consistent with no-ops or defaulted values, rather than potentially forgetting to add them later. Just a different approach. We are definitely happy to adhere to the usual style.